specification/ai/Azure.AI.Projects/agents/messages/routes.tsp

```diff
@@ -57,7 +57,7 @@ op listMessages is Azure.Core.Foundations.Operation<
     threadId: string;
 
     /** Filter messages by the run ID that generated them. */
-    @query
+    @query("run_id")
     runId?: string;
```

Comment:
please include the auto-generated swagger files in your PR, after running "npx tsv ."

Reply:
Fixed.